api/common/cloudspec/cloudspec.go

```diff
@@ -16,20 +16,21 @@ import (
 // CloudSpecAPI provides common client-side API functions
 // to call into apiserver/common/cloudspec.CloudSpec.
 type CloudSpecAPI struct {
-	facade base.FacadeCaller
+	facade   base.FacadeCaller
+	modelTag names.ModelTag
 }
 
 // NewCloudSpecAPI creates a CloudSpecAPI using the provided
 // FacadeCaller.
-func NewCloudSpecAPI(facade base.FacadeCaller) *CloudSpecAPI {
-	return &CloudSpecAPI{facade}
+func NewCloudSpecAPI(facade base.FacadeCaller, modelTag names.ModelTag) *CloudSpecAPI {
```

Comment:
I'd personally prefer we pass in a UUID since tags are supposed to be internal to the api/apiserver layers, but given we already leak them everywhere, I guess it doesn't matter.